### PR TITLE
Sepolia L1StandardBridge upgrade (devnet + testnet)

### PR DESCRIPTION
**Pull request by mdehoog**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -52,7 +52,8 @@ forge-deps:
 	forge install --no-git github.com/foundry-rs/forge-std \
 		github.com/OpenZeppelin/openzeppelin-contracts@v4.7.3 \
 		github.com/OpenZeppelin/openzeppelin-contracts-upgradeable@v4.7.3 \
-		github.com/rari-capital/solmate@8f9b23f8838670afda0fd8983f2c41e8037ae6bc
+		github.com/rari-capital/solmate@8f9b23f8838670afda0fd8983f2c41e8037ae6bc \
+		github.com/Saw-mon-and-Natalie/clones-with-immutable-args@105efee1b9127ed7f6fedf139e1fc796ce8791f2
```
Comment (anikaraghu) [inline Makefile:56]: out of curiosity, what is this used for?

File: sepolia-devnet/.env [hunk 2]
```diff
@@ -3,3 +3,6 @@ L1_RPC_URL=https://ethereum-full-sepolia-k8s-dev.cbhq.net
 L2_RPC_URL=https://base-sepolia-alpha.cbhq.net
 L1_CHAIN_ID=11155111
 L2_CHAIN_ID=11763072
+export PROXY_ADMIN=0xC5aE9023bFA79124ffA50169E1423E733D0166f1
```
Comment (anikaraghu) [inline sepolia-devnet/.env:6]: nit: by default the scripts import `../.env` so you could probably remove the `export`

**Review by anikaraghu — APPROVED** [review]
thanks!

File: Makefile (hunk 1 above)
Comment (mdehoog) [reply Makefile:56]: this is required by the latest optimism contracts (the dispute stuff uses it)

File: sepolia-devnet/.env (hunk 2 above)
Comment (mdehoog) [reply sepolia-devnet/.env:6]: unfortunately the Makefile doesn't export these, and forge only uses the `.env` file in the local dir and not the one above, so for forge scripts to read them they have to be exported (we probably should export them all for consistency)

**Pull request merged**